src/patterns/data/default/entity_attribute.tsv

```diff
@@ -4341,3 +4341,9 @@ OBA:2045294	asparaginase activity	GO:0006530	asparagine catabolic process	PATO:0
 OBA:2045295	brain connectivity attribute	UBERON:0000955	brain	PATO:0000141	structure			https://orcid.org/0000-0001-8314-2140
 OBA:2045296	defense response to other organism process attribute	GO:0098542	defense response to other organism	PATO:0001236	process quality			https://orcid.org/0000-0001-8314-2140
 OBA:2045328	DNA methylation level	GO:0006306	DNA methylation	PATO:0002062	physical quality of a process 			https://orcid.org/0000-0001-8314-2140
+OBA:2045372	gestational blood glucose level	UBERON:0009097	gravid organism	OBA:VT0000188	blood glucose amount			https://orcid.org/0000-0001-8314-2140
+OBA:2045389	leukocyte ratio	CL:0000738	leukocyte	PATO:0001470	proportionality to			https://orcid.org/0000-0001-8314-2140
+OBA:2045390	basophil ratio	CL:0000767	basophil	PATO:0001470	proportionality to			https://orcid.org/0000-0001-8314-2140
+OBA:2045391	eosinophil ratio	CL:0000771	eosinophil	PATO:0001470	proportionality to			https://orcid.org/0000-0001-8314-2140
+OBA:2045392	basophil to leukocyte ratio	CL:0000767	basophil	OBA:2045389	leukocyte ratio			https://orcid.org/0000-0001-8314-2140
+OBA:2045393	eosinophil to leukocyte ratio 	CL:0000771	eosinophil	OBA:2045389	leukocyte ratio			https://orcid.org/0000-0001-8314-2140
```

Comment:
Will you need to recreate these terms in the future using the 'ratio pattern' when it's finalized?

Reply:
> Will you need to recreate these terms in the future using the 'ratio pattern' when it's finalized?

No, they already express a ratio-type quality.